lbrynet/daemon/auth/util.py

```diff
@@ -9,21 +9,22 @@
 log = logging.getLogger(__name__)
 
 API_KEY_NAME = "api"
+LBRY_SECRET = "LBRY_SECRET"
 
 
-def sha(x):
+def sha(x: bytes) -> bytes:
     h = hashlib.sha256(x).digest()
     return base58.b58encode(h)
 
 
-def generate_key(x=None):
+def generate_key(x: bytes = None) -> bytes:
```

Comment:
Might be worth indicating this returns a base58 encoded value.  And same for sha above, as it's not generally assumed that sha hashes and keys are base58 encoded.

Reply:
I think we're going to do a types audit later and improved the typing. For now this is better than it was before :-D (before you didn't even know if it returned bytes or unicode string :scream:)